packages/filecoin-actor-utils/src/utils/logs.ts

```diff
@@ -0,0 +1,39 @@
+import { ethers } from 'ethers'
+import { DataType, Type } from '../types'
+import { ABI, abiParamsToDataType } from './abi'
+import { describeDataType } from './generic'
+
+export type RawFEVMLog = {
+  topics: Array<string>
+  data: string
+}
+
+export const describeFEVMLogs = (
+  logs: Array<RawFEVMLog>,
+  abi: ABI
+): DataType[] | null => {
+  if (!abi || !logs || logs.length === 0) return null
+
+  const iface = new ethers.utils.Interface(abi)
+  // create an array of DataTypes representing each emitted event
```

Comment:
@navFooh it seems like the sensible thing to do here is create 1 EventLogs Data type that we can use in our UI, but I wasn't sure if that actually made sense, since each EventLog is kind of its own DataType in a way. 

I'm publishing a prerelease version with this in it, just so i can get _something_ out because this works fine for now.